README.md

```diff
@@ -1,3 +1,31 @@
-# Welcome to the Gamer Club
+```-=[ fork ]=-  12/98
 
-Checkout [https://uccnetworkingsociety.github.io/Netfest-Intro/](https://uccnetworkingsociety.github.io/Netfest-Intro/)
+ |\ /\ /\ /|
+ || || || ||
+ || || || ||
+ || || || ||
+ || || || ||
+ || || || ||
+ || || || ||
+ |\_||_||_/|
+ |         |
+  \       /
+   \     /
+    |   |
+    |   |
+    |   |
+    |   |
+    |   |
+    |   |
+    |   |
+    |   |
+   /     \
+   |     |
+   |     |
+   |     |
+   |     |
+   |     |
+   |     |
+   |     |
+   \     /
+    '._.'
```

Comment:
no round bottoms

Reply:
@mloc clearly you didn't read the README documentation. Dumbo